src/workerd/api/util.h

```diff
@@ -152,6 +152,127 @@ inline kj::Promise<DeferredProxy<void>> addNoopDeferredProxy(kj::Promise<void> p
   co_return newNoopDeferredProxy();
 }
 
+// ---------------------------------------------------------
+// Deferred proxy coroutine integration
```

Comment:
Non-blocking nit: Any chance this could be moved out into its own header, and possibly moved into `src/workerd/util`? Just want to keep `util.h` itself from becoming too much Kitchen Sink.

Reply:
I agree, done in the last commit.